extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -32,10 +32,22 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 	private _copy2!: azdata.ButtonComponent;
 	private _refresh1!: azdata.ButtonComponent;
 	private _refresh2!: azdata.ButtonComponent;
+	private _onlineButton!: azdata.RadioButtonComponent;
+	private _offlineButton!: azdata.RadioButtonComponent;
+	private _modeContainer!: azdata.FlexContainer;
+	private _radioButtonContainer!: azdata.FlexContainer;
+	private _networkShareButton!: azdata.RadioButtonComponent;
+	private _blobContainerButton!: azdata.RadioButtonComponent;
+	private _originalMigrationMode!: MigrationMode;
 	private _disposables: vscode.Disposable[] = [];
 
 	constructor(wizard: azdata.window.Wizard, migrationStateModel: MigrationStateModel) {
```

Comment:
I found a bug on this page regarding the input fields not updating if the user goes backwards to the previous step and changes their migration scenario.

Bug filed: https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2126135